src/pages/game/game-won-popup/gameWonSlice.ts

```diff
@@ -0,0 +1,69 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
```

Comment:
I would move this slice to articleSlice and add isArticleRevealed state there

Reply:
I combined Article and GameWon slices. State looks like this:
data: {
        title: [],
        text: [],
    },
    isGameWon: false,
    status: LOADING_STATUS.IDLE,